### ISSUE TITLE
Batch embedding preprocessing

### ISSUE THREAD

**Issue by vaills** [bingogome/samm#74]
Hi,

I would like to segment CT scans from patients. Would it be possible to preprocess the scans to obtain all embeddings beforehand? Thank you.

Best regards,
Sunyi

**Comment by bingogome** [comment]
Yes, but probably not on the GUI. You could check the script to get the embeddings (should be only few lines), and write a new script outside of Slicer.
